docs/layout/dynamic-font-scaling.md

```diff
@@ -0,0 +1,205 @@
+
+### Android
+
+The Android Web View's font scaling mechanism is always enabled in web content and will automatically scale font sizes defined using the `px` unit. This means that any maximum or minimum font sizes specified using `px` will still be scaled even if the final font size does not align with the maximum or minimum font sizes specified.
+
+In the following example we are using the [min()](https://developer.mozilla.org/en-US/docs/Web/CSS/min) function to indicate that the font size of `.foo` should be no larger than `14px`.
+
+```css
+.foo {
+  font-size: min(1rem, 14px);
+}
+```
+
+If the root element's default font size is `16px`, and the system-level font scale is 1.5 (i.e text sizes should be increased by 50%), then `1rem` will evaluate to `24px` because `16 * 1.5 = 24`.
+
+This is larger than our defined maximum of `14px`, so one might assume that the evaluated font size of `.foo` is `14px`. However, since the Android Web View scales any font sizes defined using the `px` unit, this means the `14px` used in our `min()` function will also be scaled by 1.5.
+
+As a result, this means that the maximum computed font size is actually `21px` since `14 * 1.5 = 21` and therefore the overall computed font size of `.foo` is `21px`.
+
+### Using Modes on Different Platforms
+
+Each platform has slightly different font scaling behaviors, and the `'ios'` and `'md'` modes have been implemented to take advantage of the scaling behaviors on their respective platforms.
+
+For example, `'ios'` mode makes use of maximum and minimum font sizes to follow [Apple's Human Interface Guidelines for Typography](https://developer.apple.com/design/human-interface-guidelines/typography). `'md'` mode does not implement this same behavior because Material Design does not have that same guidance. This means that using `'md'` mode on an iOS device may allow for very large font sizes in headers and footers.
+
+As a result, we strongly recommend using `'ios'` mode on iOS devices and `'md'` mode on Android devices when using Dynamic Font Scaling.
+
+## Changing the Font Size on a Device
+
+Font scaling preferences are configured on a per-device basis by the user. This allows the user to scale the font for all applications that support this behavior. This guide shows how to enable font scaling for each platform.
+
+### iOS
+
+Font scaling on iOS can be configured in Settings > Accessibility > Display & Text Size > Larger Text. The slider on that page allows users to make the text larger or smaller across all apps that support font scaling.
+
+Users can also tap the "Large Accessibility Sizes" switch to access more font scale options.
+
+### Android
+
+Font scaling on Android can be configured in Settings > Accessibility > Display size and text. The "Font size" slider on that page allows users to make the text larger or smaller across all apps that support font scaling.
```

Comment:
This varies widely across devices; for example, on my phone it's Settings > Accessibility > Visibility enhancements > Font size and style. I also worry that we'll need to maintain these steps as OS updates come out. Maybe we should just say it can be updated in the accessibility settings and leave it at that?

Reply:
Yep that's fair. Fixed in https://github.com/ionic-team/ionic-docs/pull/3111/commits/80e48a6aa0c1d7731e399af1b2d993343ca068d1

Reply:
Should we also update the iOS description, so we don't need to worry about maintaining that either? I don't know how stable that sort of thing usually is on iPhones 😆

Reply:
Good idea: https://github.com/ionic-team/ionic-docs/pull/3111/commits/c105df8a27fceb2ec8b0d3c65754010761018f50